net.gaijin.WarThunder.yaml

```diff
@@ -30,6 +30,10 @@ modules:
         sha256: 68d4e26c170091f9ede93730f8d258e72af5f325c256ffa6e44fdb9ad0c7d638
         # curl -s -L -I "URL" | gawk -v IGNORECASE=1 '/^Content-Length/ { print $2 }'
         size: 19275822
+        x-checker-data:
+          type: rotating-url
+          url: https://warthunder.com/en/download/launcherLinux
+          pattern: 'https://cdnnow-distr.gaijinent.com/wt_launcher_linux_([0-9.]+).tar.gz'
```

Comment:
Remove all of this. It randomizes the download server which will make f-e-d-c think it has an update all the time. https://matrix.to/#/!RfXaBjokqHAbzZrgHz:matrix.org/$WrH9ulD92oRe0NEHFEuDQ9_f6kPbppug8zucZiWS-RA?via=matrix.org&via=fedora.im&via=kde.org
```suggestion
```